src/swamigui/SwamiguiSpinScale.c

```diff
@@ -210,32 +222,46 @@ swamigui_spin_scale_cb_input (GtkSpinButton *spinbutton, gdouble *newval, gpoint
   SwamiguiSpinScale *spin_scale = SWAMIGUI_SPIN_SCALE (user_data);
   GValue dispval = { 0 }, adjval = { 0 };
```

Comment:
Somewhat unrelated to your changes: The compiler complains that `dispval`, `adjval` and `spin_scale` are unused here. Is it correct that way? Could they be removed?